DEPENDENCIES.md

```diff
@@ -15,7 +15,7 @@
 | Tekton Chains                 | ghcr.io/hacbs-contract/chains/controller</br>:poc-tep-84@sha256:38e83e33b5f9561c5005870</br>a516362e219747f757662e948644fd425188db200 | Modified version of Chains installed and maintained by Pipeline Service team | Custom version required until https://github.com/tektoncd/chains/pull/436 is merged upstream                                                                              |
 | kcp                           | release-0.6                                                                                                                           | Platform - acts as a control plane for Pipeline Service                      | Upgrades follow kcp-stable releases                                                                                                                                       |
 | kind                          | 0.14.0                                                                                                                                | For local development only                                                   | Spawns a Kubernetes-in-Docker clusters. No requirement to use a particular version. Users can install the latest version available at the time                            |
-| OpenShift GitOps              | 2.3.5                                                                                                                                 | Prerequisite for managing the installation and lifecycle of components       | OpenShift GitOps uses ArgoCD at the backend                                                                                                                               |
+| OpenShift GitOps              | 1.6.0                                                                                                                                 | Prerequisite for managing the installation and lifecycle of components       | OpenShift GitOps uses Argo CD (2.4.5) at the backend                                                                                                                      |
```

Comment:
OCP and OKD have a different phrasing:
- "Red Hat OpenShift GitOps uses Argo CD to maintain cluster resources." 
- "OpenShift GitOps is an Operator that uses Argo CD as the declarative GitOps engine. "

I prefer the later one. What do you think of rephrasing to the following?
"OpenShift GitOps uses Argo CD (2.4.5) as the declarative GitOps engine"

Reply:
Done.